manifests/h/HTTPToolKit/HTTPToolKit/1.4.3/HTTPToolKit.HTTPToolKit.installer.yaml

```
@@ -0,0 +1,22 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: HTTPToolKit.HTTPToolKit
+PackageVersion: 1.4.3
+MinimumOSVersion: 10.0.0.0
+Commands:
+- httptoolkit
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```

```suggestion
  Architecture: x64
```
Try it with `x64`; the `x86` VM might be bugging out.

I had no issues installing it on Windows Sandbox & VMware Workstation.